src/routes/calendarRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+
+const { authHandler } = require("../middlewares/auth");
+const calendarController = require("../controllers/calendarController")
+
+router.post("", authHandler, calendarController.createUserCalendar);
+router.get("/user", authHandler, calendarController.getUserInfo);
+router.post("/tests", authHandler, calendarController.getTestInfo);
+router.patch("/user", authHandler, calendarController.updateCalendar);
```

Comment:
```suggestion
router.get("", authHandler, calendarController.getUserInfo);
router.post("/tests", authHandler, calendarController.getTestInfo);
router.patch("", authHandler, calendarController.updateCalendar);
```
어차피 캘린더는 user의 캘린더일테니 굳이 user를 뒤에 붙여줄 필요는 없어보입니다!